bobweb/bob/test_git_promotions.py

```diff
@@ -0,0 +1,79 @@
+from unittest import mock
+from unittest.mock import AsyncMock
+
+import pytest
+import telegram
+from asynctest import Mock
+from django.test import TestCase
+from telegram.constants import ParseMode
+from telegram.ext import CallbackContext
+
+from bobweb.bob import main, database, git_promotions
+from bobweb.bob.tests_mocks_v2 import MockBot
+
+default_commit_message = 'Title\n\n* Description <quote>text</quote>'
+
+
+async def mock_method_bad_request_error(*args, parse_mode, **kwargs):
+    if parse_mode == ParseMode.HTML:
+        raise telegram.error.BadRequest("Can't parse entities")
+
+
+@pytest.mark.asyncio
+class TestGitPromotions(TestCase):
+
+    @mock.patch('os.getenv', lambda key: default_commit_message)
+    @mock.patch('bobweb.bob.git_promotions.promote_committer_or_find_out_who_he_is', AsyncMock())
+    @mock.patch('bobweb.bob.broadcaster.broadcast')
+    async def test_broadcast_and_promote_sends_message_with_expandable_quote(self, broadcast_mock: AsyncMock):
+        bob = database.get_the_bob()
+        bob.latest_startup_broadcast_message = None
+        bob.save()
+        context = Mock(spec=CallbackContext)
+        context.bot = MockBot()
+
+        await git_promotions.broadcast_and_promote(context)
+
+        expected_message = '<blockquote expandable>Title\n\n* Description &lt;quote&gt;text&lt;/quote&gt;</blockquote>'
+        broadcast_mock.assert_called_with(
+            bot=context.bot,
+            text=expected_message,
+            parse_mode=ParseMode.HTML)
+
+        bob = database.get_the_bob()
+        self.assertEqual(default_commit_message, bob.latest_startup_broadcast_message)
+
+    @mock.patch('os.getenv', lambda key: default_commit_message)
+    @mock.patch('bobweb.bob.git_promotions.promote_committer_or_find_out_who_he_is', AsyncMock())
+    @mock.patch('bobweb.bob.database.get_the_bob', Mock())
+    @mock.patch('bobweb.bob.broadcaster.broadcast')
+    async def test_broadcast_and_promote_sends_without_expandable_quote_if_initial_sending_fails(self, broadcast_mock: AsyncMock):
```

Comment:
Do we really have to test around features like this? Can't we expect that the expendable _just works_?